RecoLocalMuon/CSCRecHitD/python/cscRecHitD_cfi.py

```diff
@@ -4,73 +4,75 @@
 
 # parameters for CSC rechit building
 from RecoLocalMuon.CSCRecHitD.cscRecHitD_cff import *
-csc2DRecHits = cms.EDProducer("CSCRecHitDProducer",
+import RecoLocalMuon.CSCRecHitD.configWireTimeWindow_cfi as _mod
+
+csc2DRecHits = _mod.configWireTimeWindow.clone(
     #
     #    Parameters for coordinate and uncertainty calculations
     #    Data and MC parameters are (still) different
     #    Needs tuning
     #
-    cscRecHitDParameters,
+    #cscRecHitDParameters, # drop this due to identical values in cfipython _mod
```

Comment:
For the same reason as we put back the other parameters in the explicit config, I would uncomment this line (and remove the comment): developers can then modify `cscRecHitDParameters` in `RecoLocalMuon/​CSCRecHitD/​python/​cscRecHitD_cff.py` without bothering about uncommenting the line here